modules/layers/src/column-layer/stroke-truncated-cone-geometry.js

```diff
@@ -0,0 +1,121 @@
+import {Geometry, uid} from '@luma.gl/core';
+
+const INDEX_OFFSETS = {
+  x: [2, 0, 1],
+  y: [0, 1, 2],
+  z: [1, 2, 0]
+};
+
+export default class TruncatedConeGeometry extends Geometry {
```

Comment:
@kaushikb11 

As suggested, you can simplify this Geometry and make it as a Cylinder, could support both stroked mode or filled mode.

Reply:
Noted. Working on it. @xintongxia